uiowa/uiowa
# [Feature] Contact info

Related to https://github.com/uiowa/uiowa/issues/8078. 



# How to test

```
ddev blt ds --site=default  && ddev drush @default.local config-split:activate contact -y && ddev drush @default.local uli    
```

```
ddev blt ds --site=now.uiowa.edu  && ddev drush @now.local config-split:activate contact -y && ddev drush @now.local uli    
```
- Naming - is `contact` good?
- Confirm empty split activates with no issues.
